api/main_endpoints/routes/officerManager.js

```diff
@@ -41,27 +41,20 @@ router.post('/submit', (req, res) => {
 
 // Find all api if email is null/undefined
 // else query by email
-router.post('/GetForm', (req, res) => {
-  if (!checkIfTokenSent(req)) {
-    return res.sendStatus(FORBIDDEN);
-  } else if (!checkIfTokenValid(req, membershipState.OFFICER)) {
-    return res.sendStatus(UNAUTHORIZED);
-  }
+router.post('/getOfficers', (req, res) => {
```

Comment:
Shouldn't this check for authorization? We don't want personal information to be publicly accessible (there are emails being returned from this endpoint).

Reply:
```suggestion
router.get('', (req, res) => {
```

Should be a get request, and the `/getOfficers` part is superfluous (at least change it to `/get` to make it consistent)

Reply:
I would also like to change line 22 to `router.put('', (req, res) => {` if possible, unless that endpoint is already in use.